### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VMCpp**
## PR Summary

Simple grammatical correction in README.md file.

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by SteveL-MSFT — DISMISSED** [review]
I don't believe this PR provides much value.  The current content is fine.

**Comment by anmenaga** [comment]
@SteveL-MSFT I actually think that PR provides more value than what is listed in the 'Files changes'. Something we discussed previously.

**Comment by RDIL** [comment]
I think this is the only one that makes more sense:

- If you already cloned but forgot to use `--recursive`, you can update submodules manually:
+ If you have already cloned but forgot to use `--recursive`, you can update submodules manually:

**Review by anmenaga — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ To make things easy, you can just clone recursively:
 git clone --recursive https://github.com/PowerShell/PowerShell.git
```
Comment (TravisEz13) [inline README.md]: `--recursive` is not needed anymore.

Do you want to fix this or merge as is?

**Review by TravisEz13 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (adityapatwardhan) [reply README.md]: I would like this to be removed as `--recursive` is not needed anymore.

**Comment by iSazonov** [comment]
@adityapatwardhan  I updated the PR description.

**Pull request merged**